platform/darwin/src/MGLGeometry.h

```diff
@@ -56,6 +74,33 @@ NS_INLINE MGLCoordinateBounds MGLCoordinateBoundsMake(CLLocationCoordinate2D sw,
     return bounds;
 }
 
+/**
+ Creates a new `MGLCoordinateQuad` structure from the given top left, top right,
+  bottom right, and bottom left coordinates .
+ */
+NS_INLINE MGLCoordinateQuad MGLCoordinateQuadMake(CLLocationCoordinate2D topLeft, CLLocationCoordinate2D topRight, CLLocationCoordinate2D bottomRight, CLLocationCoordinate2D bottomLeft) {
```

Comment:
These parameters should go in the same order as the fields are declared above, to ensure consistency between Objective-C and Swift.